docs/releases/1.10.txt

```diff
@@ -206,6 +206,9 @@ Forms
 * Form and widget ``Media`` is now served using
   :mod:`django.contrib.staticfiles` if installed.
 
+* Select widget render_options() does not take the choices argument anymore,
```

Comment:
This should be in "Backwards incompatible changes", not minor features. Is there a reason you mentioned `render_options()` but not `render()`?